weave/trace_server/trace_server_interface.py

```diff
@@ -815,6 +829,7 @@ def objs_query(self, req: ObjQueryReq) -> ObjQueryRes: ...
     def table_create(self, req: TableCreateReq) -> TableCreateRes: ...
     def table_update(self, req: TableUpdateReq) -> TableUpdateRes: ...
     def table_query(self, req: TableQueryReq) -> TableQueryRes: ...
+    def table_query_stats(self, req: TableQueryStatsReq) -> TableQueryStatsRes: ...
```

Comment:
Per the implementations, it looks like this endpoint returns the number of rows for a query.  This is surprising because `stats` suggests to me something like a mean/median/mode.  This endpoint may be more clearly named as `table_query_num_rows`

Reply:
we also have `calls_query_stats` which is similar. These are intended to support additional column stats and other summary info for a query in the future

Reply:
it will expand, yes. right now it is just `count`, but you can imagine it could contain stats per column, for example

Why is that bad?